spec/features/users_show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+describe 'Users Show', type: :feature do
+  let :user do
+    User.create(name: 'Akai321', photo: 'https://Akaiiii.jpg', bio: 'Martial Artist')
+  end
+
+  let! :post1 do
+    Post.create(author_id: user.id, title: 'The way of the Water',
+                text: 'In martial arts, the way of the water is the way of the soft and yielding')
+  end
+
+  let! :post2 do
+    Post.create(author_id: user.id, title: 'Amaterasu',
+                text: 'Amaterasu is the goddess of the sun and the universe, the ultimate source of all life')
+  end
+
+  let! :post3 do
+    Post.create(author_id: user.id, title: 'Legacy', text: 'The legacy of the righteous is a blessing')
+  end
+
+  before { visit user_path(user) }
+
+  scenario "I can see the user's profile picture" do
```

Comment:
- [ ] Great job including all the requested tests :clap: However, I noticed that some of your tests are currently failing. So kindly fix those as well so that all your tests can pass as expected :100: 
- Reference photo
![image](https://github.com/IsmailChangezi/blog-app-new/assets/102699958/730fb343-7278-452e-8399-86cc1854ad2d)

Reply:
@NahnahAJ We had to submit individually but I submitted my partner's project in my dashboard, How can I fix my mistake?